### PR TITLE
requirements numbering & assignment

### PR DESCRIPTION
**Pull request by merlin217**

### CODE REVIEW

**Comment by merlin217** [comment]
> I think we should keep assigning the tasks to notion and slack (as in not include the names). Same for account settings page details. What do you think?

Yeah agreed! I'll update on the branch. Do you think we should link each bullet point to the corresponding notion task or just keep as is?

**Comment by RamitKataria** [comment]
I think we can just keep it as is since the notion page is not public

**Comment by merlin217** [comment]
> I think we can just keep it as is since the notion page is not public

sounds good. I'll add all requirement assignments we have onto notion just in case we forget them

**Comment by merlin217** [comment]
@RamitKataria  Also just noticed during proof-reading, in the last meeting we moved stretch goals 2, 3, 4 into standard goals 4, 7, 8. This means we will have 9 standard goals & 3 stretch. I'm wondering if #4 should really be considered standard if the notification is done on email? Or are we thinking about something else?

**Comment by RamitKataria** [comment]
This should be easier than the email verification one so I think it's alright for this to be standard. The one that was about automatically sending an email once everyone has entered their availability was the hard one.

**Comment by merlin217** [comment]
> This should be easier than the email verification one so I think it's alright for this to be standard. The one that was about automatically sending an email once everyone has entered their availability was the hard one.

Updated!

File: README.md [hunk 1]
```diff
@@ -35,27 +35,36 @@ Allow users to set up meetings schedules & automatically fill in availability by
...
+8. As an attendee, I want to be able to log in to my account through the same external authentication service
+9. As a meeting organizer, I want to be able to remove a user from a meeting
+
+### Stretch Requirements
+
+1. All meeting attendees should be able to vote for the best meeting time from all the options provided by the website
+2. As the meeting organizer, I want to be able to limit access to the meeting info and availability to registered users with given email addresses
+3. As a user, I want to be able to import my availability from a link to my personal availability ICS file
+
+### Account Settings Page
```
Comment (RamitKataria) [inline README.md:60]: Maybe remove the details about the account settings page as well?

Comment (merlin217) [reply README.md]: done!

**Review by RamitKataria — APPROVED** [review]

**Pull request merged**